README.md

```diff
@@ -56,17 +56,17 @@ We organize webpack CLI as a multi-package repository using [lerna](https://gith
 Supporting developers is an important task for webpack CLI.
 Thus, webpack CLI provides different commands for many common tasks.
 
-- `build|bundle|b [entries...] [options]` - Run webpack (default command, can be omitted).
+- [`build|bundle|b [entries...] [options]`](https://webpack.js.org/api/cli/#build) - Run webpack (default command, can be omitted).
 - [`configtest|t [config-path]`](./packages/configtest/README.md#webpack-cli-configtest) - Validate a webpack configuration.
-- `help|h [command] [option]` - Display help for commands and options.
+- [`help|h [command] [option]`](https://webpack.js.org/api/cli/#help) - Display help for commands and options.
 - [`init|create|new|c|n [generation-path] [options]`](./packages/generators/INIT.md#webpack-cli-init) - Create a new webpack project.
 - [`info|i [options]`](./packages/info/README.md#webpack-cli-info) - Returns information related to the local environment.
```

Comment:
Maybe this too? Ideally we should provide link on all commands

Reply:
They are actually linked to the package README, but I think links to webpack.js.org will be better. I will update 👍🏻